File: Suiki/app/src/main/java/com/suiki/suiki/Model/BddModel/FicheMedical.java
package com.suiki.suiki.Model.BddModel;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.lang.reflect.Array;
import java.util.ArrayList;
import java.util.Date;

/**
 * Created by Massil on 11/01/2017.
 */

public class FicheMedical implements Serializable{
    @SerializedName("date")
    public Date date;
    @SerializedName("pathologie")
    public Pathologie pathologie;
    @SerializedName("symptome")
    public ArrayList<Symptome> symptomes;
    @SerializedName("phase")
    public Phase phase;
    @SerializedName("exercice")
    public ArrayList<Exercice> exercices;
}

File: Service/model/suikiBdd/ficheMedical.js
/**
 * Created by Massil on 25/01/2017.
 */
var db = require('./../../config/db');

var Schema = db.Schema;

var ficheMedical = Schema({
    date : 'Date',
    pathologie : {type : Schema.Types.ObjectId , ref : 'Pathologie'},
    symptome : [{type : Schema.Types.ObjectId , ref:'Symptome'}],
    phase : {type : Schema.Types.ObjectId , ref : 'Phase'},
    exercice : [{type : Schema.Types.ObjectId , ref:'Exercice'}]
});

module.exports = db.model('FicheMedical' , ficheMedical);